### PR TITLE
Create brians_brain.py

### PR DESCRIPTION
**Pull request by Wrrikk**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: cellular_automata/brians_brain.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+"""
```
[the remark this answers is not in the document]
Comment (Wrrikk) [reply cellular_automata/brians_brain.py:1]: handled the errors

**Review by Wrrikk — COMMENTED** [review]
Updated

**Review by Wrrikk — COMMENTED** [review]
updated

**Review by Wrrikk — COMMENTED** [review]
updated

**Review by Wrrikk — COMMENTED** [review]
updated

**Review by Wrrikk — COMMENTED** [review]
updated

**Review by Wrrikk — COMMENTED** [review]
updated

**Comment by tianyizheng02** [comment]
Closing because builds for this PR have failed for two weeks and the code hasn't been fixed

**Pull request closed without merging**